src/main/java/emu/grasscutter/server/packet/recv/HandlerGetPlayerTokenReq.java

```diff
@@ -113,7 +113,7 @@ public void handle(GameSession session, byte[] header, byte[] payload) throws Ex
         if (req.getKeyId() > 0) {
             var encryptSeed = session.getEncryptSeed();
             try {
-                var cipher = Cipher.getInstance("RSA/ECB/PKCS1Padding");
+                var cipher = Cipher.getInstance("RSA/ECB/OAEPWithSHA-256AndMGF1Padding");
```

Comment:
Did you verify that this is compatible with the client or just apply suggestions from sonarcloud?
Because this has a high chance of breaking the ps. Same for the other crypto algorithm changes.